src/Toolkit/Toolkit.WPF/UI/Controls/PopupViewer/PopupViewer.Theme.xaml

```diff
@@ -2,8 +2,136 @@
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
     xmlns:internal="clr-namespace:Esri.ArcGISRuntime.Toolkit.Internal"
-    xmlns:controls="clr-namespace:Esri.ArcGISRuntime.Toolkit.UI.Controls">
-  <Style TargetType="{x:Type controls:PopupViewer}" >
+    xmlns:controls="clr-namespace:Esri.ArcGISRuntime.Toolkit.UI.Controls"
+    xmlns:converter="using:Esri.ArcGISRuntime.Toolkit.Internal"
+    xmlns:primitives="clr-namespace:Esri.ArcGISRuntime.Toolkit.Primitives">
+
+    <internal:VisibilityConverter x:Key="PopupViewerVisibilityConverter"/>
+    <internal:HtmlToPlainTextConverter x:Key="PopupViewerHtmlToPlainTextConverter"/>
+    <Style TargetType="TextBlock" x:Key="PopupViewerHeaderStyle">
+        <Setter Property="FontSize" Value="16" />
+        <Setter Property="FontWeight" Value="Bold" />
+        <Setter Property="TextWrapping" Value="Wrap" />
+    </Style>
+    <Style TargetType="TextBlock" x:Key="PopupViewerTextStyle">
+        <Setter Property="FontSize" Value="14" />
+        <Setter Property="TextWrapping" Value="Wrap" />
+    </Style>
+    <Style TargetType="TextBlock" x:Key="PopupViewerTitleStyle">
+        <Setter Property="FontSize" Value="16" />
+        <Setter Property="TextWrapping" Value="Wrap" />
+    </Style>
+    <Style TargetType="TextBlock" x:Key="PopupViewerCaptionStyle">
+        <Setter Property="FontSize" Value="12" />
+        <Setter Property="TextWrapping" Value="Wrap" />
+    </Style>
+    <Style TargetType="{x:Type primitives:TextPopupElementView}" >
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="{x:Type primitives:TextPopupElementView}">
+                    <Grid Visibility="{Binding Text, Converter={StaticResource PopupViewerVisibilityConverter}}" >
+                        <TextBlock DataContext="{TemplateBinding Element}" 
+                                   Text="{Binding Text, Converter={StaticResource PopupViewerHtmlToPlainTextConverter}, Mode=OneWay}"
+                                   Style="{StaticResource PopupViewerTextStyle}"/>
+                        <!--<RichTextBox x:Name="TextArea" IsReadOnly="True" BorderThickness="0" />-->
```

Comment:
Did you mean to keep this placeholder?

Reply:
Yup this one is for you for the html->xaml piece